Orange/widgets/data/owselectrows.py

```diff
@@ -809,5 +880,73 @@ def resizeEvent(self, QResizeEvent):
         self.set_text()
 
 
+class DateTimeWidget(QDateTimeEdit):
+    def __init__(self, parent, col_idx, datetime_format):
+        QDateTimeEdit.__init__(self, parent)
+
+        self.parent = parent
+        self.format = datetime_format
+        self.have_date, self.have_time = datetime_format[0], datetime_format[1]
+        self.column = parent.data[:, col_idx]
```

Comment:
I'd be happier if `column` was passed as an argument, so the widget wouldn't need to assume how the parent stores data.